tc/core/cuda/cuda_tc_executor.h

```diff
@@ -37,9 +37,14 @@ class CudaTcExecutor : public TcExecutor<CudaBackend> {
   /// No tensor-related information can be checked so it is the user's
   /// responsibility to ensure that shapes and strides match. If the user
   /// doesn't then segfault will likely occur.
+  /// Pass by const reference is good for GPU now, but it may be a problem
```

Comment:
Please drop this sentence: `Pass by const reference is good for GPU now, but it may be a problem`